fluentd/datadog_checks/fluentd/fluentd.py

```diff
@@ -3,10 +3,14 @@
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
 
+import re
+import shlex
```

Comment:
This does not exist on Windows

Reply:
[Good point!](https://stackoverflow.com/questions/33560364/python-windows-parsing-command-lines-with-shlex)

It seems `get_subprocess_output()` reprocesses the `command` anyways, e.g. if passing a string it will parse it into a list of arguments:

https://github.com/DataDog/integrations-core/blob/347298ef096b88bcced0b6e7d397ef97ae8359cd/datadog_checks_base/datadog_checks/base/utils/subprocess_output.py#L46-L54

I'll refactor to simply provide the `--version` command as a string, then.

Reply:
The `shlex` usage was inspired by how the [gunicorn](https://github.com/DataDog/integrations-core/blob/master/gunicorn/datadog_checks/gunicorn/gunicorn.py) integration does things — and it currently still uses `shlex`. So I guess it might be worth going around the code base to remove `shlex` usage!